### PR TITLE
Feature/efood 0001

### PR DESCRIPTION
**Pull request by PedroTeixeira-dev**
creating the put and delete endpoints

### CODE REVIEW

File: src/config/app.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 def create_app(environment_name: str):
     # Init flask openapi3
```
Comment (cristiantcjti) [inline src/config/app.py]: @PedroTeixeira-dev If you want you can remove these comments 😉

File: src/config/routes.py [hunk 2]
```diff
@@ -3,8 +3,7 @@
 # from src.authors.api import author_api
```
Comment (cristiantcjti) [inline src/config/routes.py]: @PedroTeixeira-dev If you are not using this imports, you can remove them.

File: src/core/models.py [hunk 3]
```diff
@@ -6,12 +8,12 @@
...
+    id = db.Column(db.UUID(), primary_key=True, default=uuid.uuid4)
     created_at = db.Column(db.DateTime(), default=datetime.now)
     updated_at = db.Column(db.DateTime(), default=datetime.now, onupdate=datetime.now)
 
-    def save(self, **kwargs):
-        model = self.__class__(name=self.name)
-        db.session.add(model)
-        db.session.commit()
-        return model
+#    def save(self, **kwargs):
```
Comment (cristiantcjti) [inline src/core/models.py:15]: @PedroTeixeira-dev We can run some tests to try to make this work. Leave this code here for future tests.

File: src/custumers/api.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+
+@custumers_api.get("/custumers")
+def get_custumers():
+    return [{"id": custumer.id, "name": custumer.name, "cep": custumer.cep, "email": custumer.email} for custumer in Custumer.query.all()]
+
+
+@custumers_api.post("/custumers")
+def create_custumer(body: CustumersPostSchema):
+    print("#############", body)
```
Comment (cristiantcjti) [inline src/custumers/api.py]: @PedroTeixeira-dev You can remove this print here 👍🏻

File: src/custumers/api.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        uf=body.uf,
+        city=body.city,
+        street=body.street,
+        number=body.number,
+        complement=body.complement).save()
+    return {"id": custumer.id, "name": custumer.name}
+
+
+@custumers_api.delete("/custumers")
+def delete_custumer(body: CustumerDeleteSchema):
```
Comment (cristiantcjti) [inline src/custumers/api.py:36]: 👏🏻

File: src/custumers/api.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    print(f"Received request to update customer with email: {email}")
+
+    custumer = Custumer.query.filter_by(email=email).first()
+    if not custumer:
+        print("Customer not found")
+        return {"error": "Customer not found"}, 404
+
+    print(f"Updating customer: {custumer}")
+
+    try:
```
Comment (cristiantcjti) [inline src/custumers/api.py:61]: @PedroTeixeira-dev Great idea, we can make better use of try except throughout our code. Well done 😉 .

File: src/custumers/models.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+        self.city = city
+        self.street = street
+        self.number = number
+        self.complement = complement
+
+    def __repr__(self):
+        return f"<id {self.id}>"
+
+    def save(self, **kwargs):
+        model = self.__class__(
```
Comment (cristiantcjti) [inline src/custumers/models.py:31]: @PedroTeixeira-dev We will refactor this method but for now you can keep it.

File: src/config/app.py (hunk 1 above)
Comment (PedroTeixeira-dev) [reply src/config/app.py]: done

File: src/config/routes.py (hunk 2 above)
Comment (PedroTeixeira-dev) [reply src/config/routes.py]: done

File: src/core/models.py (hunk 3 above)
Comment (PedroTeixeira-dev) [reply src/core/models.py:15]: ok

File: src/custumers/api.py (hunk 4 above)
Comment (PedroTeixeira-dev) [reply src/custumers/api.py]: done

File: src/custumers/api.py (hunk 6 above)
Comment (PedroTeixeira-dev) [reply src/custumers/api.py:61]: :)

File: src/custumers/models.py (hunk 7 above)
Comment (PedroTeixeira-dev) [reply src/custumers/models.py:31]: ok

**Comment by PedroTeixeira-dev** [comment]
I did all the alterations as requested.

**Review by cristiantcjti — APPROVED** [review]

**Comment by cristiantcjti** [comment]
You can merge 👏🏻

**Pull request merged**